ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__multi-nightly.yaml

```diff
@@ -79,6 +79,38 @@ tests:
     - ref: ipi-install-heterogeneous
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-aws-ipi
+- as: azure-ipi-baselinecaps-v412-p2-f14
+  cron: 11 10 14,29 * *
+  steps:
+    cluster_profile: azure-qe
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:multi-latest
+    env:
+      ADDITIONAL_WORKER_VM_TYPE: Standard_D4ps_v5
+      BASE_DOMAIN: qe.azure.devcluster.openshift.com
+      E2E_RUN_TAGS: '@heterogeneous and @azure-ipi and @network-ovnkubernetes and
+        not @fips'
+      TAG_VERSION: '@4.12'
+    test:
+    - ref: ipi-install-heterogeneous
+    - chain: openshift-e2e-test-qe
+    workflow: cucushift-installer-rehearse-azure-ipi-baselinecaps-v412
+- as: azure-ipi-disconnected-fullyprivate-p2-f14
```

Comment:
@jianlinliu remove this as not supported on 4.12.

Reply:
I saw we have a ci profile - `04_heterogeneous_Disconnected IPI on Azure & fully Private Cluster & OVN & x86_arm64_workers`, is that true we dose not support it?

Reply:
yes, it is. That was an in-advance testing done to gather some infos.

The blocking aspect of disconnected on heterogeneous clusters is the support for sparse manifest lists in order to allow the customers to mirror only a subset of the manifests (the ones for the arches they actually want).

However, as that effort seems still far from landing in production, we might want to enable disconnected support for 4.14 as is (users will need to mirror the whole payload as we usually do for testing now). 

Similar concerns are being tracked for the operators catalog that will increase a lot the magnitude of the required storage to support multi-arch clusters.

Reply:
Thanks for the explaination.